src/firefly/voltmeters.py

```diff
@@ -71,6 +71,20 @@ async def update_devices(self, registry):
             # Connect the details button signal
             details_slot = partial(self.details_window_requested.emit, ic.name)
             row.details_button.clicked.connect(details_slot)
+        # Remove old shutters from the combobox
+        for idx in range(self.ui.shutter_combobox.count()):
+            self.ui.shutter_combobox.removeItem(idx)
+        # Add the shutters to the shutter combobox
+        shutters = registry.findall("shutters", allow_none=True)
```

Comment:
I would like explicitly remove 'front end shutter' from the list because it happens very often that ppl accidentally choose front end shutter.

Reply:
I agree that the FES should not be available.

Instead of hard-coding the name "front_end_shutter", though, I used the existing `allow_close` and `allow_open` attributes on the shutter devices. The shutter will only be present if both are not `False`. A corollary is that shutters without those attributes will still be allowed.